documentation/src/main/asciidoc/topics/con_client_interoperability.adoc

```diff
@@ -0,0 +1,87 @@
+[id='storage_formats_client_interop-{context}']
+= Storage Formats and Client Interoperability
+HTTP and Hot Rod protocols are designed to handle data in different formats.
+For example, Hot Rod is designed to send and receive binary data while REST
+clients are best suited for handling text-based formats such as JSON, XML, or
+plain text.
+
+By configuring the MediaType for keys and values, you describe the data format
+of the cache. {brandname} can then convert data on the fly during cache
+operations.
+
+If you want to use the {brandname} Console, Hot Rod clients, and REST clients
+interchangeably when operating on data, you should configure caches to use the
+Protobuf storage format.
+
+Storing data in other formats affects client interoperability and {brandname}
+capabilities, as follows:
+
+.Protobuf Storage
+[%header,cols=2*]
+|===
+2+| Protobuf storage is suitable for...
+| {brandname} Console
+| Yes
+| REST clients
+| Yes
+| Java Hot Rod clients
+| Yes
+| Non-Java Hot Rod clients
+| Yes
+| {brandname} Search
+| Yes
+| Custom Java objects
+| Yes
+|===
+
+.Text-Based Storage
+[%header,cols=2*]
+|===
+2+| Text-based storage formats are suitable for...
+| {brandname} Console
+| Yes
+| REST clients
+| Yes
+| Java Hot Rod clients
+| Yes
+| Non-Java Hot Rod clients
+| No
```

Comment:
I am having second thoughts on this. If the cache stores a random text in UTF-8, it should be readable from other clients such as node.js and C#. So this is a Y